content/en/docs/guides/status-codes.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Status Codes"
+description: >-
+     Explains the status codes used in gRPC.
+---
+
+### Overview
+All RPCs started at a client return a `status` object composed of an integer
+code and a string error description. The server-side chooses the status it returns
+for a given RPC. Applications should only use values defined below. 
+
+The gRPC client and server-side implementations may generate and
+return status on their own when errors happen. Only a subset of
```

Comment:
Is this clearer?

"When an error situation occurs, the gRPC library may produce a corresponding status code. The library may do this either on the client- or the server-side."

Reply:
Incorporated with "status code" replaced by "`status`"